index.html

```diff
@@ -9,7 +9,7 @@
 <body>
   <!-- Navigation start -->
   <nav>
```

Comment:
Modify this navigation section as the main heading of the page. A page should always have basic header and footer.

Reply:
A header tag is added to cover the navigation section